kbcmd/cmdlib/args/property_test.go

```diff
@@ -20,6 +20,9 @@ func (e TestEnum) IsValid() bool {
 	return e == TestEnumFOO
 }
 
+type testObj2 struct {
```

Comment:
Leftover from debugging?

Reply:
Indeed... removed it in https://github.com/killbill/kbcli/pull/36/commits/19f6f1ec70b986fd2d52e47b33cffaaf1d4cebd2